packages/fxa-auth-server/config/index.ts

```diff
@@ -150,13 +150,37 @@ const conf = convict({
       format: String,
       doc: 'Apple auth client secret',
     },
+    keyId: {
+      default: '',
+      env: 'APPLE_AUTH_KEY_ID',
+      format: String,
+      doc: 'Apple auth key id',
+    },
     redirectUri: {
       default:
         'https://localhost.dev:3030/post_verify/third_party_auth/callback',
       env: 'APPLE_AUTH_REDIRECT_URI',
       format: String,
       doc: 'Apple auth redirect uri',
     },
+    privateKey: {
+      default: '',
+      env: 'APPLE_AUTH_PRIVATE_KEY',
+      format: String,
+      doc: 'Apple auth private key',
+    },
+    teamId: {
+      default: '',
+      env: 'APPLE_AUTH_TEAM_ID',
+      format: String,
+      doc: 'Apple auth team id',
+    },
+    appleEndpoint: {
```

Comment:
This is actually the openid spec `aud`.

From https://developer.apple.com/documentation/sign_in_with_apple/generate_and_validate_tokens:

> The audience registered claim identifies the intended recipient of the client secret. Since the client secret is sent to the validation server, use https://appleid.apple.com.

 I think we can put it as a constant in file and not as a config.